src/ASM/ASMs1DLag.C

```diff
@@ -7,7 +7,7 @@
 //!
 //! \author Einar Christensen / SINTEF
 //!
-//! \brief Driver for assembly of 1D Lagrange FE models.
+//! \brief Driver for assembly of 1D %Lagrange FE models.
```

Comment:
what does this do?

Reply:
Without it, doxygen will insert a hyperlink to the class with that name (Lagrange).